app/src/main/res/layout/manage_proguard.xml

```diff
@@ -154,31 +167,30 @@
                 <ImageView
                     android:layout_width="32dp"
                     android:layout_height="32dp"
-                    android:layout_marginRight="8dp"
+                    android:layout_marginStart="8dp"
```

Comment:
don't do that when the rest of the app doesn't respect RTL

Reply:
done